site/static/linux/opensuse/tetrate-getenvoy.repo

```diff
@@ -0,0 +1,15 @@
+[tetrate-getenvoy-stable]
+name=Tetrate GetEnvoy - Stable
+baseurl=https://tetrate.bintray.com/getenvoy-rpm/rhel/$releasever/$basearch/stable/
```

Comment:
We should have use a fixed $releasever, like `8` here, because SUSE have different versioning semantics. can you test if this repo file works with `opensuse/leap` docker image?

Reply:
fixed. And checked it works.